staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -1134,6 +1164,14 @@ func ValidateCustomResourceDefinitionOpenAPISchema(schema *apiextensions.JSONSch
 	return allErrs
 }
 
+func validatePath(schema *apiextensions.JSONSchemaProps, path string) bool {
+	if ss, err := structuralschema.NewStructural(schema); err == nil {
```

Comment:
what is this logic? Surprising to read without explanation.

Reply:
Here is in consideration of leveraging the same field path checking func and for code maintainance. Raised https://github.com/kubernetes/kubernetes/pull/118041#discussion_r1265978138

Reply:
I think it was more a comment about the relationship between the structural schema parsing and the field path validation. A follow-up to update the comment to explain that we only support field paths we can verify map to structural stanzas would be fine, non-blocking for this PR